MPXPLAY/AU_CARDS/AU_CARDS.C

```diff
@@ -23,9 +23,9 @@
 #include "mpxplay.h"
 #include "dmairq.h"
 #ifndef SBEMU
-#include "newfunc\dll_load.h"
-#include "au_mixer\au_mixer.h"
-#include "display\display.h"
+#include "newfunc/dll_load.h"
```

Comment:
The MPXPlay source conteained a lot of file paths with backslashes as directory separators instead of forward slashes. Fixing that was a requirement to get this to work on Linux.

@crazii  However, there is still the matter of inconsistencies in terms of mixing upper-case and lower-case letters in file names throughout the source code, which I worked around for the time being by mounting a FAT32 image and building the source code in there, since the workflow is executed in a Linux environment.

However, such a FAT32-workaround would be made unnecessary if we applied consistent naming of both the files in the Git repo and the references to those files within the sources (preferably all files and paths consistently in lower-case, as is customary in source code). This would touch a lot of files at once, but shouldn't change ore complicate anything otherwise. I recommend such a bulk-rename (with the corresponding changes in file and path references throughout the source code) to be done in a separate commit, though. So not combined with any other changes.

Reply:
Fixed in merge https://github.com/crazii/SBEMU/pull/28